### PR TITLE
partial pep8 cleanup in plot and plot3d

### PR DESCRIPTION
**Pull request by fchapoton**
just cleaning up plot3d for pep8 compliance

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/plot/plot3d/parametric_plot3d.py [hunk 1]
```diff
@@ -989,7 +989,7 @@ def g(x, y): return x, y+sin(y), x**2 + y**2
     if isinstance(f, Vector):
         f = tuple(f)
 
-    if isinstance(f, (list, tuple)) and len(f) > 0 and isinstance(f[0], (list, tuple)):
+    if isinstance(f, (list, tuple)) and f and isinstance(f[0], (list, tuple)):
         return sum([parametric_plot3d(v, urange, vrange, plot_points=plot_points, **kwds) for v in f])
```
Comment (dcoudert) [inline src/sage/plot/plot3d/parametric_plot3d.py]: may be you can change to `sum(parametric_plot3d(v, urange, vrange, plot_points=plot_points, **kwds) for v in f)` on the way ?

File: src/sage/plot/plot3d/plot3d.py [hunk 2]
```diff
@@ -191,7 +191,7 @@ def __init__(self, dep_var, indep_vars):
         """
         all_vars = sage_getargspec(self.transform).args[1:]
         if set(all_vars) != set(indep_vars + [dep_var]):
-            raise ValueError('variables were specified incorrectly for this coordinate system; incorrect variables were %s' % list(set(all_vars).symmetric_difference(set(indep_vars+[dep_var]))))
+            raise ValueError('variables were specified incorrectly for this coordinate system; incorrect variables were %s' % list(set(all_vars).symmetric_difference(set(indep_vars + [dep_var]))))
```
Comment (dcoudert) [inline src/sage/plot/plot3d/plot3d.py]: can we split this long line ?

File: src/sage/plot/plot3d/plot_field3d.py [hunk 3]
```diff
@@ -143,13 +143,14 @@ def plot_vector_field3d(functions, xrange, yrange, zrange,
...
+            def cm(x):
+                return colors
 
     max_len = max(v.norm() for v in vectors)
-    scaled_vectors = [v/max_len for v in vectors]
+    scaled_vectors = [v / max_len for v in vectors]
 
     if center_arrows:
-        G = sum([plot(v, color=cm(v.norm()), **kwds).translate(p-v/2) for v, p in zip(scaled_vectors, points)])
+        G = sum([plot(v, color=cm(v.norm()), **kwds).translate(p - v / 2) for v, p in zip(scaled_vectors, points)])
```
Comment (dcoudert) [inline src/sage/plot/plot3d/plot_field3d.py]: is th `[...]` necessary for this sum ?

**Review by dcoudert — COMMENTED** [review]
otherwise, LGTM.

**Review by dcoudert — APPROVED** [review]
LGTM.

**Comment by vbraun** [comment]
Merge conflict

**Comment by fchapoton** [comment]
back to positive after trivial merge

**Comment by dcoudert** [comment]
the linter is not happy

**Comment by fchapoton** [comment]
the linter issue was fixed in #39095

**Comment by dcoudert** [comment]
then, back to positive.

**Pull request merged**